### PR TITLE
Add EIP: Non-Fungible Key Bound Token

### PR DESCRIPTION
**Pull request by KBTStandardAdmin**
A standard interface for Non-Fungible Key Bound Tokens, also known as a Non-Fungible **KBT**

The Key Bound Token Standard is a new standard focused on self-custodial security. The users of the non-fungible version can benefit from its integrated decentralized anti-theft mechanism. By binding two additional wallets, this standard prevents attackers and scammers alike to use your asset, even if they gain control of your seed phrase or private key.

### CODE REVIEW

File: EIPS/eip-6809.md [hunk 1]
```diff
@@ -0,0 +1,683 @@
...
+author: Mihai Onila (@MihaiORO), Nick Zeman (@NickZCZ), Narcis Cotaie (@NarcisCRO)
+discussions-to: https://ethereum-magicians.org/t/non-fungible-key-bound-token-kbt/13625
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-03-31
+requires: 721
+---
+
+A standard interface for Non-Fungible Key Bound Tokens, also known as a Non-Fungible **KBT**[^1].
```
Comment (Pandapip1) [inline EIPS/eip-6809.md:14]: @SamWilsn - why was this not flagged by EIPw?
```suggestion
```

**Review by Pandapip1 — CHANGES_REQUESTED** [review]

File: EIPS/eip-6809.md (hunk 1 above)
Comment (NickZCZ) [reply EIPS/eip-6809.md:14]: Hi @Pandapip1, we can remove the 'standard' keyword if that's the issue. Can you guide us on how to proceed with the Source URLs in the footnotes?

**Review by sgilflores3 — DISMISSED** [review]

File: EIPS/eip-6809.md (hunk 1 above)
Comment (Pandapip1) [reply EIPS/eip-6809.md:14]: No, it's that there's text outside of any section. There must not be text outside of any section.

**Comment by Pandapip1** [comment]
@eth-bot rerun

File: EIPS/eip-6809.md [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+
+We considered **NFKBTs** being used by every individual who wishes to add additional security to their non-fungible assets, as well as consignment to third-party wallets/brokers/banks/insurers/museums. **NFKBTs** allow tokens to be more resilient to attacks/thefts, by providing additional protection to the asset itself on a self-custodial level.
+
+## Motivation
+
+TIn this fast-paced technologically advancing world, people learn and mature at different speeds. The goal of global adoption must take into consideration the target demographic is of all ages and backgrounds. Unfortunately for self-custodial assets, one of the greatest pros is also one of its greatest cons. The individual is solely responsible for their actions and adequately securing their assets. If a mistake is made leading to a loss of funds, no one is able to guarantee their return.
```
Comment (SamWilsn) [inline EIPS/eip-6809.md:26]:
```suggestion
In this fast-paced technologically advancing world, people learn and mature at different speeds. The goal of global adoption must take into consideration the target demographic is of all ages and backgrounds. Unfortunately for self-custodial assets, one of the greatest pros is also one of its greatest cons. The individual is solely responsible for their actions and adequately securing their assets. If a mistake is made leading to a loss of funds, no one is able to guarantee their return.
```

**Review by NarcisCRO — APPROVED** [review]
Hi @SamWilsn
We applied the same fixes as for 6808 and we're ready for merge as well.

File: EIPS/eip-6809.md [hunk 3]
```diff
@@ -0,0 +1,522 @@
...
+
+9. **Environmentally Friendly:** Since the security features are coded into the **NFKBT**, there is no need for centralized servers, shipping, or the production of physical object/s. Thus leading to a minimal carbon footprint by the use of **NFKBTs**, working hand in hand with Ethereum’s change to a _PoS_[^14] network.
+
+10. **User Experience:** The security feature can be activated by a simple call to the [addBindings](#addbindings-function) function. The user will only need two other wallets, which will act as `_keyWallet1`[^15] and `_keyWallet2`[^16], to gain access to all of the benefits **NFKBTs** offer. The optional security features improve the overall user experience and Ethereum ecosystem by ensuring a safety net for those who decide to use it. Those that do not use the security features are not hindered in any way. This safety net can increase global adoption as people can remain confident in the security of their assets, even in the scenario of a compromised wallet.
+
+## Specification
+
+### ERC-N (Token Contract)
```
Comment (SamWilsn) [inline EIPS/eip-6809.md:58]:
```suggestion
### `IKBT721` (Token Contract)
```

**Review by SamWilsn — APPROVED** [review]

**Review by NarcisCRO — APPROVED** [review]

**Pull request merged**